storage/rocksdb/CMakeLists.txt

```diff
@@ -50,6 +50,22 @@ IF(HAVE_SCHED_GETCPU)
   ADD_DEFINITIONS(-DHAVE_SCHED_GETCPU=1 -DROCKSDB_SCHED_GETCPU_PRESENT)
 ENDIF()
 
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.2")
+SET(CMAKE_REQUIRED_FLAGS ${CMAKE_CXX_FLAGS})
+INCLUDE(CheckCXXSourceCompiles)
+CHECK_CXX_SOURCE_COMPILES("
+#include <cstdint>
+#include <nmmintrin.h>
+int main() {
+  volatile uint32_t x = _mm_crc32_u32(0, 0);
+}
+" HAVE_SSE42)
+IF (HAVE_SSE42)
+  ADD_DEFINITIONS(-DHAVE_SSE42)
+ELSE ()
+  MESSAGE(${MYROCKS_STATUS_MODE} "No SSE42 support found. Not building MyRocks")
```

Comment:
Is "Not building MyRocks" message true?

Reply:
In our case we couldn't stop building it. We still need to build it but without -DHAVE_SSE42
In general I hope all our build machines should support it.

Reply:
OK, then the message should read "Building MyRocks without SSE 4.2 support"

Reply:
It ties into the -DWITH_ROCKSDB=0|1|2 functionality here. It will either be a warning and skip build of MyRocks or it will be a failure.